src/engraving/dom/score.h

```diff
@@ -582,14 +582,14 @@ class Score : public EngravingObject, public muse::Injectable
 
     bool isShowInvisible() const { return m_showInvisible; }
     bool showUnprintable() const { return m_showUnprintable; }
-    bool showFrames() const { return m_showFrames; }
+    bool showBorders() const { return m_showBorders; }
```

Comment:
This method is about something different; it is about the score elements called "Frame". So it should not be renamed.

Reply:
Not to be confused with the next line, which is about page borders ;-)